ChangeLog

```diff
@@ -1,328 +1,332 @@
---------------------
-- get_storage_group_rep() has been refactored and marked for deprecation,
-  please use alternative function get_storage_group_srdf_details()
-- get_storage_group_rep_list() has been refactored and marked for deprecation,
-  please use alternative function get_replication_enabled_storage_groups()
-- get_storagegroup_snapshot_list() has been refactored and marked for
-  deprecation, please use alternative function
-  get_storage_group_snapshot_list()
-- create_storagegroup_snap() has been refactored and marked for deprecation,
-  please use alternative function create_storage_group_snap()
-- get_storagegroup_snapshot_generation_list() has been refactored and marked
-  for deprecation, please use alternative function
-  get_storage_group_snapshot_generation_list()
-- modify_storagegroup_snap() has been refactored and marked for deprecation,
-  please use alternative function modify_storage_group_snapshot()
-- delete_storagegroup_snapshot() has been refactored and marked for
-  deprecation, please use alternative function delete_storage_group_snapshot()
-- choose_snapshot_from_list_in_console() has marked for deprecation
-- is_vol_in_rep_session() has been refactored and marked for deprecation,
-  please use alternative function is_volume_in_replication_session()
-- are_vols_rdf_paired() has been refactored and marked for deprecation,
-  please use alternative function are_volumes_rdf_paired()
-- get_storagegroup_srdfg_list() has been refactored and marked for deprecation,
-  please use alternative function get_storage_group_srdf_group_list()
-- get_storagegroup_srdf_details() has been refactored and marked for
-  deprecation, please use alternative function get_storage_group_rdf_details()
-- create_storagegroup_srdf_pairings() has been refactored and marked for
-  deprecation, please use alternative function
-  create_storage_group_srdf_pairings()
-- modify_storagegroup_srdf() has been refactored and marked for deprecation,
-  please use alternative function modify_storage_group_srdf()
-- suspend_storagegroup_srdf() has been refactored and marked for deprecation,
-  please use alternative function suspend_storage_group_srdf()
-- establish_storagegroup_srdf() has been refactored and marked for deprecation,
-  please use alternative function establish_storage_group_srdf()
-- failover_storagegroup_srdf() has been refactored and marked for deprecation,
-  please use alternative function failover_storage_group_srdf()
-- failback_storagegroup_srdf() has been refactored and marked for deprecation,
-  please use alternative function failback_storage_group_srdf()
-- delete_storagegroup_srdf() has been refactored and marked for deprecation,
-  please use alternative function delete_storage_group_srdf()
-
-PyU4V/utils/console.py
-----------------------
-- choose_from_list() has been marked for deprecation, please implement any
-  console driven commands from your own scripts and not from PyU4V
+PyU4V Change Log
+================
+
```

Comment:
You need to add change log details about the changes in this pull request

Reply:
I put it down the end.  I will bring it up